src/async_fuse/memfs/s3_metadata.rs

```diff
@@ -797,7 +829,7 @@ impl<S: S3BackEnd + Sync + Send + 'static> MetaData for S3MetaData<S> {
         flags: u32,
     ) -> DatenLordResult<usize> {
         let data_len = data.len();
-        let (result, _, _) = {
+        let (result, _) = {
```

Comment:
If we don't need this varaible anymore , we should do refactor here.

Reply:
There're some functions with the similar issue above. We should take a look. Well maybe not this PR, it deserves a separate PR.

Reply:
Ok, I will refactor such functions when I finished the tasks at hands